src/NzbDrone.Core/Localization/Core/en.json

```
@@ -485,6 +485,8 @@
   "DownloadClientQbittorrentSettingsContentLayoutHelpText": "Whether to use qBittorrent's configured content layout, the original layout from the torrent or always create a subfolder (qBittorrent 4.3.2+)",
   "DownloadClientQbittorrentSettingsFirstAndLastFirst": "First and Last First",
   "DownloadClientQbittorrentSettingsFirstAndLastFirstHelpText": "Download first and last pieces first (qBittorrent 4.1.0+)",
+  "DownloadClientQbittorrentSettingsPropagateTags": "Propagate Tags",
```

I think `Add Series Tags` would be a better name.

```suggestion
  "DownloadClientQbittorrentSettingsAddSeriesTags": "Add Series Tags",
```